spec/helpers/application_helper_spec.rb

```diff
@@ -1641,4 +1641,23 @@
       ).to eq('host_services')
     end
   end
+
+  describe "#multiple_relationship_link" do
+    context "When record is a Container Provider" do
```

Comment:
> shouldn't there be also a test that checks that when user arrives to provider summary page there's a 200 OK?

@abonas Good call. I've added a spec for `ems_middleware` somewhat identical to `ems_container` that addresses the rendering of the `show` template. Also verified that the spec will fail with the same error that was reported in #8803 without the fix in `application_helper`
```
[----] F, [2016-05-23T10:59:28.093808 #78690:3fde55c5e1f8] FATAL -- : Error caught: [ActionView::Template::Error] undefined method `manageiq_providers_middleware_manager_path' for #<#<Class:0x007fbcb>
```